### PR TITLE
Cardboard as a material

### PR DESCRIPTION
**Pull request by RarkGrames**
#### Summary
```SUMMARY: Content "Implementing cardboard as a material"```

#### Purpose of change
Implementing cardboard as a material, distinct from paper.

#### Describe the solution
- Adding cardboard as a material
- Adding cardboard repair to misc repair kit
- Adding cardboard as an item
- Replacing paper with cardboard in items
- Replacing paper with cardboard in recipes

#### Testing
- Spawned some cardboard, checked the weight/volume per charge against paper
- Spawned some cardboard items to check the uncraft recipes
- Checked out the crafting recipes that use cardboard

#### Additional context
This PR started out as an effort to improve the `fletching` crafting requirement, but implementing a separate material is a better overall solution.

### CODE REVIEW

**Comment by Brian-Otten** [comment]
Nice addition to this might be a recipe for making cardboard (if the survivor really wants too), since it's quite doable.
https://www.wikihow.com/Make-Cardboard

**Comment by RarkGrames** [comment]
That's not a really good solution, because it transforms the material into an other one.

**Comment by I-am-Erk** [comment]
I like it, I don't know if cardboard quantities should be the same as paper though. I picture a useful piece of cardboard being significantly larger than a piece of paper.

Dino Dave approves this PR MOST ENTHUSIASTICALLY.

**Comment by Night-Pryanik** [comment]
But... cardboard is just dense paper, ain't it?

**Comment by RarkGrames** [comment]
Surprisingly paper is more dense than cardboard, it's just that paper is produced in thinner sheets ([paper](https://www.aqua-calc.com/page/density-table/substance/paper-coma-and-blank-standard), [cardboard](https://www.aqua-calc.com/page/density-table/substance/cardboard)).

So a given volume of paper is heavier than the same volume of cardboard (4 vs 2 density in CDDA terms).

**Comment by RarkGrames** [comment]
> 
> 
> I like it, I don't know if cardboard quantities should be the same as paper though. I picture a useful piece of cardboard being significantly larger than a piece of paper.

Do you mean to for example double the volume and drop the charges to half?

**Comment by RarkGrames** [comment]
> 
> 
> But... cardboard is just dense paper, ain't it?

The main purpose was to incorporate all the cardboard materials into the fletching recipe, excluding paper. I couldn't find anything solid about paper fletching (only one, that said that the paper gets destroyed after one shot). But I've found some references to people actually using cardboard.

**Comment by I-am-Erk** [comment]
> > I like it, I don't know if cardboard quantities should be the same as paper though. I picture a useful piece of cardboard being significantly larger than a piece of paper.
> 
> Do you mean to for example double the volume and drop the charges to half?

Yeah, double or more. I'd even say like x10, unless there are a bunch of recipes calling for cardboard(1)

**Comment by RarkGrames** [comment]
I increased the volume/weight of one charge five times and changed around everything according to that. Additionally matched the weight of the uncraft yields to the weight of the boxes.

File: data/json/requirements/materials.json [hunk 1]
```diff
@@ -151,10 +151,9 @@
       [
         [ "feather", 10 ],
         [ "duct_tape", 40 ],
-        [ "paper", 40 ],
+        [ "cardboard", 10 ],
```
Comment (I-am-Erk) [inline data/json/requirements/materials.json]:
```suggestion
        [ "cardboard", 2 ],
```
A single paper cup produces 2 cardboard and should have enough for an arrow or two. A small cardboard box should be able to fletch several arrows, and a cardboard box large enough to hide in should make tons of them.

Comment (RarkGrames) [reply data/json/requirements/materials.json]: That recipe produces 10 arrows.

Comment (I-am-Erk) [reply data/json/requirements/materials.json]: Ah, okay. It can probably stay at the current requirement then.

**Pull request merged**